### PR TITLE
Copy editing for Chapter 2

### PR DESCRIPTION
**Pull request by smfoote**

### CODE REVIEW

File: manuscript/02-Functions.md [hunk 1]
```diff
@@ -303,7 +303,7 @@ console.log(doAnotherThing.name);       // "doAnotherThing"
 
 In this code, `doSomething()` has a `name` property equal to `"doSomething"` because it's a function declaration. The anonymous function expression `doAnotherThing()` has a `name` of `"doAnotherThing"` due to the variable to which it is assigned.
 
-While function declarations and function expressions as in the last example are easy to find an appropiate name for, ECMAScript 6 goes further to ensure that all functions have appropriate names:
+While appropriate names for function declarations and function expressions, as in the last example, are easy to find, ECMAScript 6 goes further to ensure that all functions have appropriate names:
```
Comment (nzakas) [inline manuscript/02-Functions.md]: Too many commas with this change, makes the sentence unwieldy.

Comment (smfoote) [inline manuscript/02-Functions.md]: What if we take out `"as in the last example"`? I'd be happy to revert, but I found this sentence a bit gnarly as I was reading it.

Comment (nzakas) [inline manuscript/02-Functions.md]: That works.

**Comment by smfoote** [comment]
Diff is updated. How does it look?

**Comment by nzakas** [comment]
Looks good, thanks!

**Pull request merged**